fs-base/src/main/java/com/palantir/giraffe/file/MoreFiles.java

```diff
@@ -493,71 +492,102 @@ public static boolean isEmpty(Path path) throws IOException {
     }
 
     /**
-     * Given a {@code Path} representing a directory, retrieve the list of files
-     * and directories contained in that {@code Path} as a {@code List}. The
-     * {@code Path} objects are obtained as if by resolving the name of the
-     * directory objects against the given {@code Path}.
+     * Gets a list of the files and directories in the given directory. The
+     * returned paths are {@linkplain Path#resolve(Path) resolved} against the
```

Comment:
This is mostly based on the documentation for `Files#newDirectoryStream` which is the source of the conventions for all of these methods.

In general, I use `@link` when I'm discussing a specific type or method as its own entity and `@linkplain` when I'm discussing a concept that is implemented by or relates to a specific type or method. There's also the fact that `@link` always uses monospaced font which looks weird when you are linking a word, rather than a name.